Changelog.md

```diff
@@ -12,6 +12,7 @@ Compiler Features:
  * SMTChecker: Add constraints to better correlate ``address(this).balance`` and ``msg.value``.
  * SMTChecker: Support the ``value`` option for external function calls.
  * Commandline Interface: Disallowed the ``--experimental-via-ir`` option to be used with Standard Json, Assembler and Linker modes.
+ * Yul Optimizer: Fix a crash in LoadResolver, when ``keccak256`` had particular arguments.
```

Comment:
```suggestion
 * Yul Optimizer: Fix a crash in LoadResolver when ``keccak256`` has non-identifier arguments.
```

Reply:
But not every non-identifier arguments? Maybe I'll change to "has particular non-identifier arguments"?

Reply:
Changed it. Writing this changelog for this was more work than the fix :)